cmd/eksctl-anywhere/cmd/downloadartifacts.go

```diff
@@ -70,32 +71,47 @@ func downloadArtifacts(context context.Context, opts *downloadArtifactsOptions)
 	}
 	bundlesManifestUrl := release.BundleManifestUrl
 
-	specManifests := []string{
-		bundlesManifestUrl,
-		clusterSpec.GetReleaseManifestUrl(),
-	}
 	reader := files.NewReader(files.WithUserAgent(fmt.Sprintf("eks-a-cli-download/%s", version.Get().GitVersion)))
-	for _, manifestURI := range specManifests {
-		if opts.dryRun {
-			logger.Info(fmt.Sprintf("Found artifact: %s\n", manifestURI))
-			continue
-		}
-		if err = downloadArtifact("", opts.downloadDir, manifestURI, reader); err != nil {
-			return fmt.Errorf("error downloading artifact: %v", err)
+
+	// download the eks-a-release.yaml
+	if !opts.dryRun {
+		releaseManifestURL := clusterSpec.GetReleaseManifestUrl()
+		if err := downloadArtifact(filepath.Join(opts.downloadDir, filepath.Base(releaseManifestURL)), releaseManifestURL, reader); err != nil {
+			return fmt.Errorf("error downloading release manifest: %v", err)
 		}
 	}
 
-	bundle := clusterSpec.VersionsBundle
-	for component, manifestList := range bundle.Manifests() {
-		for _, manifest := range manifestList {
-			if opts.dryRun {
-				logger.Info(fmt.Sprintf("Found artifact: %s\n", manifest.URI))
-				continue
-			}
-			if err = downloadArtifact(component, opts.downloadDir, manifest.URI, reader); err != nil {
-				return fmt.Errorf("error downloading artifact for component %s: %v", component, err)
+	versionBundles := clusterSpec.Bundles.Spec.VersionsBundles
+	for i, bundle := range versionBundles {
```

Comment:
If we want to have the manifests for all the versions bundles, then we need a subfolder per version bundle. Different VersionBundle's can have different versions of the same component manifest.

Reply:
added a subfolder per version, updated the PR desc to include what that'll look like